pkg/options/client_opts.go

```diff
@@ -29,9 +29,9 @@ import (
 type ClientOpts struct {
 	APIConfig      api.Config
 	DefaultTimeout int
+	Client         *api.Client
```

Comment:
I don't think this is necessary.  If you need this just call the GetClient() method.